packages/core/src/lib/register.ts

```diff
@@ -58,6 +58,9 @@ export const registerWithUsernameAndPassword = async (
   });
 
   await assignRegistrationResult(ctx, provider, id);
+  ctx.userLog.userId = id;
```

Comment:
currently we don't have registration logs, we can handle this later in https://linear.app/silverhand/project/core-logs-451d2cd0dc86

Reply:
an issue added for user registration log ([LOG-1512](https://linear.app/silverhand/issue/LOG-1512/log-register)). these user logs can help us to make more sense of users' preferences. what do you think? @logto-io/eng

Reply:
you are right, but user logs is beyond current project.

Reply:
let's keep this as a reminder and fix user logs functionality later.